File: trail1/두 수의 짝홀 여부/parity-of-two-numbers.cs
using System;

public class Codetree
{  
    public static void Main()
    {
        string[] arr = Console.ReadLine().Split();
        var(a, b) = (int.Parse(arr[0]), int.Parse(arr[1]));
        string val1 = a%2==0 ? "even" : "odd";
        string val2 = b%2==0 ? "even" : "odd";
        Console.WriteLine(val1);
        Console.WriteLine(val2);
    }
}

File: trail1/두 수의 짝홀 여부/parity-of-two-numbers.cpp
#include <iostream>
#include <string>
using namespace std;

int main() {
    int a, b;
    cin >> a >> b;
    string val1 = a%2==0 ? "even" : "odd";
    string val2 = b%2==0 ? "even" : "odd";
    cout << val1 << "\n" << val2 << endl;
    return 0;
}